pom.xml

```diff
@@ -107,7 +107,7 @@
 			<dependency>
 				<groupId>javax.servlet</groupId>
 				<artifactId>jstl</artifactId>
-				<version>1.1.2</version>
+				<version>1.2</version>
```

Comment:
That is a standard upgrade,, we always upgrade standard upgrades like 1.2.0, so kindly make it standard thanks

Reply:
+1 yes I missed that

Reply:
Push again and squash the changes

Reply:
i updted it but it faille to build ,on the maven repository it is 1.2 not 1.2.0  
https://mvnrepository.com/artifact/javax.servlet.jsp.jstl/jstl
i don' t know

Reply:
should i update the version 1.2?

Reply:
<!-- https://mvnrepository.com/artifact/javax.servlet.jsp.jstl/jstl -->
<dependency>
    <groupId>javax.servlet.jsp.jstl</groupId>
    <artifactId>jstl</artifactId>
    <version>1.2</version>
</dependency>

Reply:
@achilep yeah it seems they have changed the versioning convention this time. 1.2 should be the one. @sherrif10 what do you think? If it is a problem we might have to stay with 1.1.2.

Reply:
Since the description in the ticket id say upgrading to 1.2, then lets leave it there

Reply:
This could be changed to use the newer [JSTL artifact](https://search.maven.org/artifact/javax.servlet.jsp.jstl/javax.servlet.jsp.jstl-api/1.2.2/jar).